### ISSUE TITLE
[Fix] Unable to find droplet image identifier

### ISSUE THREAD

**Issue by sharma-shruti** [digitalocean/doctl#784]
**Describe the Issue:**
While creating a droplet, I was looking for Ubuntu 18.04.3 (LTS) x64 image identifier using ```doctl compute image list``` command but the output string displays nothing. I had to search in some existing comments to find the exact naming string.

```bash 
$ doctl compute image list
ID    Name    Type    Distribution    Slug    Public    Min Disk
$ 
```

Please correct me if I am doing something wrong here.

**Environment:**

 - doctl version: `1.41.0-release`
 - OS: `Ubuntu 18.04.3 (LTS) x64`
 - Installation method: `snap install doctl`

**Additional Details:**

**Comment by bentranter** [comment]
Hey @sharma-shruti, thanks for the report.

By default, `doctl compute images list` will show only private images. If you don't have any private images, you won't see any output.

If you want to see public images, you'll need to pass the `--public` flag:

```sh
doctl compute image list --public
```

This will returns all images, both public and private, so that should return the results you're expecting.

---

Sorry that you ran into this issue – it looks like the default behaviour of showing only private images isn't the most user-friendly approach. I've raised this issue internally, so we'll see if there's a way to make this experience better.

**Comment by sharma-shruti** [comment]
Thanks, @bentranter. This works for me. Appreciate your response.

**Comment by bentranter** [comment]
Closing this as resolved based on #785
